esmvalcore/iris_helpers.py

```diff
@@ -358,3 +359,121 @@ def has_unstructured_grid(cube: Cube) -> bool:
     if cube.coord_dims(lat) != cube.coord_dims(lon):
         return False
     return True
+
+
+# List containing special cases for unit conversion. Each list item is another
+# list. Each of these sublists defines one special conversion. Each element in
+# the sublists is a tuple (standard_name, units). Note: All units for a single
+# special case need to be "physically identical", e.g., 1 kg m-2 s-1 "equals" 1
+# mm s-1 for precipitation
+_SPECIAL_UNIT_CONVERSIONS = [
+    [
+        ("precipitation_flux", "kg m-2 s-1"),
+        ("lwe_precipitation_rate", "mm s-1"),
+    ],
+    [
+        ("equivalent_thickness_at_stp_of_atmosphere_ozone_content", "m"),
+        ("equivalent_thickness_at_stp_of_atmosphere_ozone_content", "1e5 DU"),
+    ],
+]
```

Comment:
where are these units taken from? Are they standard CMOR - if so we should check against the CMOR table, also - if we need to add more of these special vars and units, do we have to edit the script in place, or wouldn't it be a wee bit better if we had these in a separate file? Sorry, too many questions packed into one single comment :smile:

Reply:
You managed to comment on one of the few lines that were just copy-pasted from another place (this was necessary to avoid circular imports) 😄 

https://github.com/ESMValGroup/ESMValCore/blob/f622e408e9fb9919c69870dc52b52cddc8b0767f/esmvalcore/preprocessor/_units.py#L18-L32

These have been added in https://github.com/ESMValGroup/ESMValCore/pull/1574 and https://github.com/ESMValGroup/ESMValCore/pull/2270. I don't think this list will get awfully long, so I think it's fine to leave it like that.

Reply:
hahaha thanks, Manu! Well, am an old man, and memory (whether it be physical, L2 or RAM) is low on my Intel Celeron brain :rofl: